MyWeight/Extensions/UITableViewCell+ReuseIdentifier.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  UITableViewCell+ReuseIdentifier.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 22/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import UIKit
+
+extension UITableViewCell {
```

Comment:
Ok, you convinced me about the protocol, but I have a suggestion to the signature for the `register` methods, will comment there ;-)